R/ab_scenario.R

```diff
@@ -197,6 +238,54 @@ ab_json = function(
   json_r
 }
 
+#' Convert JSON representation of trips from A/B Street into an 'sf' object
+#'
+#' This function takes a path to a JSON file representing an A/B Street
+#' scenario, or an R representation of the JSON in a list, and returns
+#' an `sf` object with the same structure as objects returned by
+#' [ab_scenario()].
+#'
+#' Note: the departure time in seconds is divided by 10000 on conversion
+#' to represent seconds, which are easier to work with that 10,000th of
+#' a second units.
+#'
+#' @param json Character string or list representing a JSON file or list that
+#'   has been read into R and converted to a data frame.
+#'
+#' @return An `sf` data frame
```

Comment:
Maybe slightly more context on the return value e.g. An 'sf' data frame containing an A/B Street scenario

Reply:
Good point. Now updated to 

```
#' @return An `sf` data frame representing travel behaviour scenarios
#'   from, and which can be fed into, A/B Street. Contains the following
#'   columns: person (the ID of each agent in the simulation),
#'   departure (seconds after midnight of the travel starting),
#'   mode (the mode of transport, being `Walk`, `Bike`, `Transit` and `Drive`),
#'   purpose (what the trip was for, e.g. `Work`), and
#'   geometry (a linestring showing the start and end point of the trip/stage).
```